plugins/modules/instance_interface_settings.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+"""Implementation for the linode.cloud.instance_interface_settings module."""
```

Comment:
I was thinking about implementing this in the `instance` resource, but given it requires interface IDs and functionally redundant with `default_route` I figured it'd make more sense to make it a separate module.